x/ibc-rate-limit/contracts/rate-limiter/src/contract_tests.rs

```diff
@@ -1,17 +1,18 @@
 #![cfg(test)]
 
 use crate::packet::Packet;
+use crate::state::rbac::Roles;
 use crate::{contract::*, test_msg_recv, test_msg_send, ContractError};
 use cosmwasm_std::testing::{mock_dependencies, mock_env, mock_info};
-use cosmwasm_std::{from_binary, Addr, Attribute, Uint256};
+use cosmwasm_std::{from_binary, Addr, Attribute, MessageInfo, Uint256};
 
 use crate::helpers::tests::verify_query_response;
-use crate::msg::{InstantiateMsg, PathMsg, QueryMsg, QuotaMsg, SudoMsg};
-use crate::state::tests::RESET_TIME_WEEKLY;
-use crate::state::{RateLimit, GOVMODULE, IBCMODULE, RATE_LIMIT_TRACKERS};
-
-const IBC_ADDR: &str = "IBC_MODULE";
-const GOV_ADDR: &str = "GOV_MODULE";
+use crate::msg::{InstantiateMsg, MigrateMsg, PathMsg, QueryMsg, QuotaMsg, SudoMsg};
+use crate::state::flow::tests::RESET_TIME_WEEKLY;
+use crate::state::storage::{GOVMODULE, IBCMODULE, RATE_LIMIT_TRACKERS, RBAC_PERMISSIONS};
+use crate::state::rate_limit::RateLimit;
+const IBC_ADDR: &str = "osmo1vz5e6tzdjlzy2f7pjvx0ecv96h8r4m2y92thdm";
+const GOV_ADDR: &str = "osmo1tzz5zf2u68t00un2j4lrrnkt2ztd46kfzfp58r";
```

Comment:
In a follow-up PR, lets write a doc for telling people how to copy/paste the cosmwasm code and update this variable